pyart/util/radar_utils.py

```diff
@@ -0,0 +1,184 @@
+    radar.sweep_mode['data'] = np.array(['vertical_pointing'] * nsweeps)
+    radar.fixed_angle['data'] = np.ones(nsweeps, dtype='float32') * 90.0
+    radar.sweep_start_ray_index['data'] = np.arange(nsweeps, dtype='int32')
+
+    if radar.instrument_parameters is not None:
+        for key in ['prt_mode', 'follow_mode', 'polarization_mode']:
+            if key in radar.instrument_parameters:
+                ip_dic = radar.instrument_parameters[key]
+                ip_dic['data'] = np.array([ip_dic['data'][0]] * nsweeps)
+
+    # Attributes that do not need any changes
+    # radar.altitude
+    # radar.altitude_agl
+    # radar.latitude
+    # radar.longitude
+
+    # radar.range
+    # radar.ngates
+    # radar.nrays
+
+    # radar.metadata
+    # radar.radar_calibration
+
+    # radar.time
+    # radar.fields
+    # radar.antenna_transition
+    # radar.scan_rate
+    return
+
+
+def join_radar(radar1, radar2):
+    """
+    Combine two radar instances into one.
+
+    Parameters
+    ----------
+    radar1 : Radar
+        Radar object.
+    radar2 : Radar
+        Radar object.
+    """
+    # must have same gate spacing
+    new_radar = copy.deepcopy(radar1)
+    new_radar.azimuth['data'] = np.append(radar1.azimuth['data'],
+                                          radar2.azimuth['data'])
+    new_radar.elevation['data'] = np.append(radar1.elevation['data'],
+                                            radar2.elevation['data'])
+
+    if len(radar1.range['data']) >= len(radar2.range['data']):
+        new_radar.range['data'] = radar1.range['data']
+    else:
+        new_radar.range['data'] = radar2.range['data']
+
+    # to combine times we need to reference them to a standard
+    # for this we'll use epoch time
+    estring = "seconds since 1970-01-01T00:00:00Z"
+    r1dt = num2date(radar1.time['data'], radar1.time['units'])
+    r2dt = num2date(radar2.time['data'], radar2.time['units'])
+    r1num = datetime_utils.datetimes_from_radar(radar1, epoch=True)
+    r2num = datetime_utils.datetimes_from_radar(radar2, epoch=True)
+    new_radar.time['data'] = np.append(r1num, r2num)
+    new_radar.time['units'] = datetime_utils.EPOCH_UNITS
+
+
+    for var in new_radar.fields.keys():
+        sh1 = radar1.fields[var]['data'].shape
+        sh2 = radar2.fields[var]['data'].shape
+        new_field = np.ma.zeros([sh1[0] + sh2[0],
+                                max([sh1[1], sh2[1]])]) - 9999.0
+        new_field[0:sh1[0], 0:sh1[1]] = radar1.fields[var]['data']
+        new_field[sh1[0]:, 0:sh2[1]] = radar2.fields[var]['data']
+        new_radar.fields[var]['data'] = new_field
+
+    # radar locations
+    # TODO moving platforms - any more?
+    if (len(radar1.latitude['data']) == 1 &
+        len(radar2.latitude['data']) == 1 &
+        len(radar1.longitude['data']) == 1 &
+        len(radar2.longitude['data']) == 1 &
+        len(radar1.altitude['data']) == 1 &
+        len(radar2.altitude['data']) == 1):
```

Comment:
The indenting is off here if you want to fix it.  It was also off in the original case so this is not as important.